tools/coav-control/parser.cc

```diff
@@ -245,6 +248,10 @@ control_options parse_cmdline(int argc, char *argv[])
                 exit(-EINVAL);
             }
 
+        // Port
+        } else if (p.option == "-p" || p.option == "--port") {
+            opts.port = (unsigned int) stoul(p.val, nullptr, 0);
```

Comment:
why do you set parameters 2 and 3 here? do we need to support port numbers in anything other than base 10?